.github/workflows/ci.yml

```diff
@@ -7,9 +7,15 @@ on:
   pull_request:
     branches:
       - master
+
 jobs:
-  test-ubuntu-latest:
-    runs-on: ubuntu-latest
+  test-ubuntu:
+    strategy:
+      matrix:
+        ubuntu:
+          - 16.04
```

Comment:
I believe YAML will treat this as a floating-point value rather than a string. But it looks like it works OK.

Reply:
As long as it works we don't need to change it.